src/mame/machine/genfruit.h

```diff
@@ -0,0 +1,33 @@
+// license:BSD-3-Clause
+// copyright-holders:SomeRandomGuyIdk
+#ifndef GENFRUIT_H_
+#define GENFRUIT_H_
+
+#include "sound/samples.h"
+
+#define SAMPLE_METER 0
+#define SAMPLE_PAYOUT 1
+
+const char *const genfruit_sample_names[] =
```

Comment:
You shouldn’t be using preprocessor macros when it isn’t absolutely necessary, and you shouldn’t be putting things in the global namespace unnecessarily.

Reply:
Fixed.